app.js

```diff
@@ -0,0 +1,82 @@
+const bookCollection = document.querySelector('.bookshelf');
+const addButton = document.querySelector('.submit');
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+
+const books = [];
+class BookCollection {
```

Comment:
- [ ] [REQUIRED] Good job so far!  It uses JavaScript classes instead of `objects` and also `arrays`, please you should have implement `objects` and `arrays` inside the `classes` 🙏🏼